modules/cnv-add-nic-to-vm.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-edit-vms.adoc
+// * cnv/cnv_users_guide/cnv-editing-vm-template
+//
+// Establishing conditionals so content can be re-used for editing VMs
+// and VM templates.
+
+ifeval::["{context}" == "cnv-edit-vms"]
+:cnv-vm-add-nic:
+:object: virtual machine
+:object-gui: Virtual Machines
+endif::[]
+
+ifeval::["{context}" == "cnv-editing-vm-template"]
+:cnv-vm-add-nic-template:
+:object: virtual machine template
+:object-gui: Virtual Machine Templates
+endif::[]
+
+[id="cnv-vm-add-nic_{context}"]
+
+= Adding a network interface to a {object}
+
+.Procedure
+
+. From the *{object-gui}* tab, select the {object} you created.
+. Select the *Network Interfaces* tab.
+. Click *Add NIC*.
+. In the *Add NIC* window, specify the *Name*, *Model*, *Network*, *Type*,
+and *MAC Address* of the network interface.
+. For the virtual machine to be accessible, it must be restarted. Click *Add NIC* to add the network interface without restarting the virtual
+machine.
```

Comment:
This seems non-intuitive:
_it must be restarted. Click *Add NIC* to add the network interface without restarting_
The VM must be restarted but don't restart it?

Reply:
Hi @mburke5678 - This is a limitation in the current design, or seems to be. There was a button planned that auto-restarted the VM, but it's coming in another release. 

For now, I have changed the wording slightly to: Click Add Network Interface to add the network interface without restarting the virtual
machine. Note that for the virtual machine to be accessible, it must be restarted.